h/api/models/document.py

```diff
@@ -26,17 +26,18 @@ class Document(Base, mixins.Timestamps):
     def __repr__(self):
         return '<Document %s>' % self.id
 
+    @property
+    def title(self):
+        meta = DocumentMeta.query.filter(DocumentMeta.document_id == self.id,
+                                         DocumentMeta.type == 'title').first()
+        if meta:
+            return meta.value
+
     @classmethod
-    def find_or_create_by_uris(cls, claimant_uri, uris,
```

Comment:
Fixed for both `find_by_uris` and `find_or_create_by_uris`.

What do you think about `api.storage`, should it be passed-in there as well? Especially now that we're querying the database in there.